css/home.css

```diff
@@ -131,17 +143,141 @@
   backdrop-filter: blur(10px);
   border-radius: 10px;
   margin: 10% auto;
+  position: relative;
+  align-items: center;
+  display: flex;
+  justify-content: center;
 }
 
-.domain .col-6 .web,
+.web {
+  width: 389px;
+  height: 79px;
+  left: -3px;
+  top: 7px;
+  background: linear-gradient(
+    180deg,
+    rgb(158 204 189 / 54%) 0%,
+    rgb(197 225 218) 100%
+  );
+  backdrop-filter: blur(10px);
+  border-radius: 10px;
+  margin: 10% auto;
+  position: relative;
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
+.block {
+  width: 389px;
```

Comment:
Did you check the responsiveness of the component for which this CSS has been written? I would suggest using `%` instead of `px` as the unit. Or else you can also try using `@media` query if it is not responsive.